practice.js

```diff
@@ -1,14 +1,26 @@
+//Importing all the common nodes needed throughout the code
 let input = document.getElementById("input");
 let div = document.getElementById("result");
+let add = document.getElementById("add");
+let clearButton = document.getElementById('clear-button');
 
+//Function to handle disable attribute of add button
+const addChangeHandler = (e) => {
+    e ? add.removeAttribute('disabled') : add.setAttribute('disabled', 'true');
+}
+
+//Functionality for adding a task
 const addTask = () => {
+
+    // Importing all required nodes
     let list = document.getElementById("list");
     let tasks = list.children.length;
     let newTask = document.createElement('li');
```

Comment:
Use const if the there is not reassignment to this variables.

Reply:
This applies to all the places where let is being used.

Reply:
Changed let to const where it is required.